Enhance Test Coverage for @yamada-ui/layouts

txxxxc opened the pull request:
<!---
Thanks for creating a Pull Request 💖!

Please read the following before submitting:
- PRs that adds new external dependencies might take a while to review.
- Keep your PR as small as possible.
- Limit your PR to one type (docs, feature, refactoring, ci, or bugfix)
- If a PR is not merged within one week of its creation, core members may intervene.
-->

Closes #786 

## Description

Enhance Test Coverage for @yamada-ui/layouts

## Is this a breaking change (Yes/No):

No

## Additional Information

To keep the cost per review small, we have not written tests for the [ZStack](https://github.com/yamada-ui/yamada-ui/blob/e620bd0fa5c7c39501c50c020fcd6fb0cdd5b447/packages/components/layouts/src/stack.tsx#L211) component.

koralle commented on packages/components/layouts/tests/layouts.test.tsx:
```diff
@@ -2,16 +2,37 @@ import { render, a11y, screen, fireEvent, waitFor } from "@yamada-ui/test"
+  })
+
+  test("renders aspect ratio correctly", () => {
+    render(
```
There's no assertion, but what is this code testing?

txxxxc replied:
My apologies. There is such a test in this file. I thought such a test would be valuable.

koralle replied:
I should be the one apologizing for the oversight. Sorry about that. Your way of thinking is not wrong.

We'll start gradually fixing similar existing test code over here as well.

Thank you!

koralle commented on packages/components/layouts/tests/layouts.test.tsx:
```diff
@@ -97,6 +118,39 @@ describe("<Divider />", () => {
 
+describe("<SimpleGrid />", () => {
+  test("renders SimpleGrid correctly", () => {
+    render(<SimpleGrid>GridSimple</SimpleGrid>)
```
There's no assertion, but what is this code testing?

koralle commented on packages/components/layouts/tests/layouts.test.tsx:
```diff
@@ -243,3 +297,22 @@ describe("<Grid />", () => {
+
+describe("<GridItem />", () => {
+  test("renders grid correctly", () => {
+    render(<GridItem>GridItem</GridItem>)
```
There's no assertion, but what is this code testing?

koralle commented on packages/components/layouts/tests/layouts.test.tsx:
```diff
@@ -97,6 +110,39 @@ describe("<Divider />", () => {
+
 describe("<Stack />", () => {
   test("renders stack correctly", () => {
     render(<Stack>Stack</Stack>)
```
There are no assertions here; what is being tested?

koralle commented:
Could you switch from `getBy*` to `findBy*`? 
We're planning to establish Lint rules for test code using [eslint-plugin-testing-library](https://github.com/testing-library/eslint-plugin-testing-library) soon, and we intend to enforce the use of `findBy*`.

koralle commented on .changeset/tender-bears-confess.md:
```diff
@@ -0,0 +1,5 @@
+---
```
It is not necessary to run `pnpm changeset` for a PR that only adds or updates tests.  
Could you remove this file, please?

See [wiki](https://github.com/yamada-ui/yamada-ui/wiki/Tips-for-Test#frequently-asked-questions)

txxxxc replied:
fixed in [5e85eee](https://github.com/yamada-ui/yamada-ui/pull/976/commits/5e85eee5e127277d4a6652c3a0dca9068e670410)

txxxxc commented:
@koralle 
Thanks for the review, I just replaced the `getBy*` method with a `findBy*` method. For now, I just modified the code I added in the previous commit, and I would like to replace the other `getBy*` methods in `layouts.test.tsx` as well. Do I have to do so?

koralle commented:
@txxxxc 

> Thanks for the review, I just replaced the getBy* method with a findBy* method. For now, I just modified the code I added in the previous commit, and I would like to replace the other getBy* methods in layouts.test.tsx as well. Do I have to do so?

You only need to fix the new assertions you've added.   
The existing `getBy*` from before you created this PR will be corrected later in one go.
